eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -55,87 +62,73 @@ Fetching an object by location. Location can be either a location ID (42) or a r
 
 ## Example: fetching an object
 
-```
-Resource: /content/object/(location_id)
-Method: GET
+__Resource__: _/content/object/<LOCATION_ID>_
```

Comment:
If it's `/content/object`, it's not a `LOCATION_ID` :)

As far as I can't tell, this resource doesn't really differ from the current one, does it ?